{"name": "test_Accounts", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "@allure.step('Browser Launched')\n    def LaunchBrowser():\n        try:\n            winName = getWindowName()\n            Browser.switch_to.window(winName[1])\n            Browser.close()\n            Browser.switch_to.window(winName[0])\n            Browser.get(\"https://tcdi-test.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=apps\")\n            sleep(5)\n            fn_objExist(Browser.find_element_by_name(\"loginfmt\"))\n            Browser.find_element_by_id(\"i0116\").send_keys(\"test.fieldsales@cditechnologies.com\")\n            sleep(5)\n            WebDriverWait(Browser,140).until(EC.element_to_be_clickable((By.ID,\"idSIButton9\")))\n            Browser.find_element_by_id(\"idSIButton9\").click()\n            rptStepDetails(\"Pass\", \"Login details entered & clicked on login button\")\n            WebDriverWait(Browser,140).until(EC.element_to_be_clickable((By.ID,\"i0118\")))\n            fn_objExist(Browser.find_element_by_id(\"i0118\"))\n            Browser.find_element_by_id(\"i0118\").send_keys(\"Tr0x@CRM\")\n            sleep(5)\n            fn_objExist(Browser.find_element_by_id(\"idSIButton9\"))\n            Browser.find_element_by_id(\"idSIButton9\").click()\n            rptStepDetails(\"Pass\",\"Clicked on sign on button\")\n            fn_objExist(Browser.find_element_by_id(\"idBtn_Back\"))\n            Browser.find_element_by_id(\"idBtn_Back\").click()\n>           sleep(10)\nE           KeyboardInterrupt\n\n..\\..\\Commonfunction.py:102: KeyboardInterrupt\n\nDuring handling of the above exception, another exception occurred:\n\n    def test_Accounts ():\n        try:\n            rptTC_Title(\"Validate Accounts navigation & verify page object \")\n>           LaunchBrowser()\n\nTC01_Accounts.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\na = (), kw = {}, __tracebackhide__ = True, params = OrderedDict(), args = []\n\n    @wraps(func)\n    def impl(*a, **kw):\n        __tracebackhide__ = True\n        params = func_parameters(func, *a, **kw)\n        args = list(map(lambda x: represent(x), a))\n        with StepContext(self.title.format(*args, **params), params):\n>           return func(*a, **kw)\n\nc:\\python36\\lib\\site-packages\\allure_commons\\_allure.py:157: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    @allure.step('Browser Launched')\n    def LaunchBrowser():\n        try:\n            winName = getWindowName()\n            Browser.switch_to.window(winName[1])\n            Browser.close()\n            Browser.switch_to.window(winName[0])\n            Browser.get(\"https://tcdi-test.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=apps\")\n            sleep(5)\n            fn_objExist(Browser.find_element_by_name(\"loginfmt\"))\n            Browser.find_element_by_id(\"i0116\").send_keys(\"test.fieldsales@cditechnologies.com\")\n            sleep(5)\n            WebDriverWait(Browser,140).until(EC.element_to_be_clickable((By.ID,\"idSIButton9\")))\n            Browser.find_element_by_id(\"idSIButton9\").click()\n            rptStepDetails(\"Pass\", \"Login details entered & clicked on login button\")\n            WebDriverWait(Browser,140).until(EC.element_to_be_clickable((By.ID,\"i0118\")))\n            fn_objExist(Browser.find_element_by_id(\"i0118\"))\n            Browser.find_element_by_id(\"i0118\").send_keys(\"Tr0x@CRM\")\n            sleep(5)\n            fn_objExist(Browser.find_element_by_id(\"idSIButton9\"))\n            Browser.find_element_by_id(\"idSIButton9\").click()\n            rptStepDetails(\"Pass\",\"Clicked on sign on button\")\n            fn_objExist(Browser.find_element_by_id(\"idBtn_Back\"))\n            Browser.find_element_by_id(\"idBtn_Back\").click()\n            sleep(10)\n            rptStepDetails(\"Pass\", \"Clicked on NO Button in the popup\")\n            WebDriverWait(Browser,240).until(EC.element_to_be_clickable((By.ID,\"AppLandingPage\")))\n            fn_objExist(Browser.find_element_by_id(\"AppLandingPage\"))\n            Browser.switch_to.frame(\"AppLandingPage\")\n            fn_objExist(Browser.find_element_by_xpath('//*[@id=\"AppDetailsSec_1_Item_1\"]/div[1]'))\n            Browser.find_element_by_xpath('//*[@id=\"AppDetailsSec_1_Item_1\"]/div[1]').click()\n            Browser.switch_to.window(winName[0])\n            sleep(20)\n            print(\"Browser Launch completed !!!!\")\n    \n            assert len(winName[0])>0\n        except:\n>           assert False\nE           AssertionError\n\n..\\..\\Commonfunction.py:115: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\n    def test_Accounts ():\n        try:\n            rptTC_Title(\"Validate Accounts navigation & verify page object \")\n            LaunchBrowser()\n            rptStepDetails(\"Pass\", \"Browser Launched Successfully\")\n            sleep(2)\n            fn_NavigateTo_AccountsPage()\n            fn_NavigateTo_QuotesPage()\n            fn_ClickOnNewQuote()\n    #         fn_VerifyQuotePage()\n            fn_EnterQuoteSummaryInfo()\n            sleep(random.randint(0,15))\n            fn_Click_Save_OnHeader()\n            print(\"completed\")\n            assert True\n        except:\n>           rptStepDetails(\"Fail\", \"Unknown error look for error in report\")\n\nTC01_Accounts.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nTestStatus = 'Fail', TestStep = 'Unknown error look for error in report'\n\n    @allure.step(\"{TestStatus},{TestStep}\")\n    def rptStepDetails(TestStatus=None,TestStep=None):\n        if TestStatus == \"Pass\" or TestStatus ==\"pass\":\n            allure.attach.file(\"C:/Users/DELL/Downloads/image.png\", \"ScreenShot\", attachment_type=None, extension=\"PNG\")\n            assert True\n        else:\n            allure.attach.file(\"C:/Users/DELL/Downloads/image.png\", \"ScreenShot\", attachment_type=None, extension=\"PNG\")\n>           assert False\nE           AssertionError\n\n..\\..\\ReportFunction.py:19: AssertionError"}, "steps": [{"name": "Browser Launched", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"c:\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\DELL\\eclipse-workspace\\CDI\\Commonfunction.py\", line 115, in LaunchBrowser\n    assert False\n"}, "steps": [{"name": "'Pass','Object exist hence exiting'", "status": "passed", "attachments": [{"name": "ScreenShot", "source": "6b08f75a-8fa3-4b31-b83a-59ec35dc164f-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Pass'"}, {"name": "TestStep", "value": "'Object exist hence exiting'"}], "start": 1599995814154, "stop": 1599995814156}, {"name": "'Pass','Login details entered & clicked on login button'", "status": "passed", "attachments": [{"name": "ScreenShot", "source": "3a004133-4b0f-4f37-90b3-80dccbca9da1-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Pass'"}, {"name": "TestStep", "value": "'Login details entered & clicked on login button'"}], "start": 1599995819496, "stop": 1599995819499}, {"name": "'Pass','Object exist hence exiting'", "status": "passed", "attachments": [{"name": "ScreenShot", "source": "ff805e33-67ea-475e-ab79-7dba3aa61cb5-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Pass'"}, {"name": "TestStep", "value": "'Object exist hence exiting'"}], "start": 1599995821765, "stop": 1599995821767}, {"name": "'Pass','Object exist hence exiting'", "status": "passed", "attachments": [{"name": "ScreenShot", "source": "51ed7174-1101-4e7c-8952-178fc968285d-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Pass'"}, {"name": "TestStep", "value": "'Object exist hence exiting'"}], "start": 1599995826918, "stop": 1599995826920}, {"name": "'Pass','Clicked on sign on button'", "status": "passed", "attachments": [{"name": "ScreenShot", "source": "9e69250c-7135-40eb-9cbb-d5899e27067c-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Pass'"}, {"name": "TestStep", "value": "'Clicked on sign on button'"}], "start": 1599995830804, "stop": 1599995830821}, {"name": "'Pass','Object exist hence exiting'", "status": "passed", "attachments": [{"name": "ScreenShot", "source": "46f30efc-6e9b-4955-abe0-b33ab934a18c-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Pass'"}, {"name": "TestStep", "value": "'Object exist hence exiting'"}], "start": 1599995830922, "stop": 1599995830925}], "start": 1599995801950, "stop": 1599995844131}, {"name": "'Fail','Unknown error look for error in report'", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"c:\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\DELL\\eclipse-workspace\\CDI\\ReportFunction.py\", line 19, in rptStepDetails\n    assert False\n"}, "attachments": [{"name": "ScreenShot", "source": "ed8365df-a740-493b-8051-f162df112ace-attachment.PNG"}], "parameters": [{"name": "TestStatus", "value": "'Fail'"}, {"name": "TestStep", "value": "'Unknown error look for error in report'"}], "start": 1599995844135, "stop": 1599995844138}], "start": 1599995801950, "stop": 1599995844142, "uuid": "60495d37-2898-4e0b-8904-17eed1ab3d9c", "historyId": "1308f9c3f65abc16844080ba636a349e", "testCaseId": "915d1631ed0ced950e6348799c47ffc5", "fullName": "scr.Accounts.TC01_Accounts#test_Accounts", "labels": [{"name": "parentSuite", "value": "scr.Accounts"}, {"name": "suite", "value": "TC01_Accounts"}, {"name": "host", "value": "GOOGLE"}, {"name": "thread", "value": "6968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scr.Accounts.TC01_Accounts"}]}